app/views/idv/in_person/state_id.html.erb

```diff
@@ -110,10 +110,23 @@
         ) %>
   </div>
   <div class="margin-bottom-5">
+    <% state_id_hint = capture do %>
+      <%= t('in_person_proofing.form.state_id.state_id_number_hint') %>
+      <% [
+           [t('in_person_proofing.form.state_id.state_id_number_hint_spaces'), ' '],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_forward_slashes'), '/'],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_asterisks'), '*'],
+           [t('in_person_proofing.form.state_id.state_id_number_hint_dashes'), '-', true],
+         ].each do |text, symbol, last| %>
+         <span class="usa-sr-only"><%= text %><%= ',' if !last %></span>
```

Comment:
the comma was probably the trickiest bit! I think leaving the comma inside each string is error prone.

if we were joining "plain" text, I would consider having us use someting like [`Array#to_sentence`](https://www.rubydoc.info/docs/rails/Array:to_sentence) or `safe_join` to handle the commas, but making sure the comma stays inside the HTML tags makes that tricky

Reply:
This is a nice solution :superb: (needs a emoji)

Reply:
This is very easy to grok. I say leave it. Seems like Array#to_sentence would require you to pass in the wrapping HTML for the conjoining elements as a param? This is just much more explicit.